src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -114,7 +128,7 @@ export class VisualizeEmbeddable
   private visCustomizations?: Pick<VisualizeInput, 'vis' | 'table'>;
   private subscriptions: Subscription[] = [];
   private expression: string = '';
-  private vis: Vis;
+  public vis: Vis;
```

Comment:
how come?

Reply:
it's needed to access the underlying vis settings when rendering the embeddables in the view events flyout. This would be reverted as part of the efforts in #4483.